pyo3-macros-backend/src/module.rs

```diff
@@ -210,7 +210,7 @@ pub fn add_fn_to_module(
             pyo3::types::PyCFunction::internal_new(
                 name,
                 doc,
-                pyo3::class::PyMethodType::PyCFunctionWithKeywords(#wrapper_ident),
+                unsafe { std::mem::transmute(#wrapper_ident as *const std::os::raw::c_void) },
```

Comment:
Transmuting to `enum` is safe?

Reply:
This is actually a function-pointer-to-function-pointer cast (I changed the arguments to `PyCFunction::internal_new`).

Actually I think I can get rid of this by refactoring `PyCFunction::internal_new` a bit more. It will have a slight user-facing change (but a good one imo) so I'll do it in a follow-up PR.